### DISCUSSION TITLE
Laravel Geocoder

### DISCUSSION THREAD

**Discussion by hofmannsven** [HassanZahirnia/laravel-package-ocean#66] [Package Suggestions]
[Laravel Geocoder](https://github.com/spatie/geocoder) is a useful package to convert any address to GPS coordinates using Google's geocoding service.

New category suggestion: Geo

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package! It will be live in a moment.

As for the new category suggestion, this is something I will decide in the future since technically I agree there could be dozens and dozens of categories that could be added, but that would make things more complicated. My goal is to keep the website simple and clean.
